vscode-extension/src/utilities/versionUpdateUtils.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Utils file for handling actions to perform whenever user
+ * installs or updates the extension.
+ * VS Code automatically checks for version updates
+ * (https://code.visualstudio.com/docs/editor/extension-marketplace#_extension-autoupdate)
+ * but I haven't been able to find a way to use this to trigger any
+ * actions. Also even though the Walkthrough documentation says
```

Comment:
Could look into vscode.extensions.onDidChange in the future but not sure if it fires on update or just add/remove/enable/disable